### PR TITLE
Turns messages into functions

### PR DESCRIPTION
**Pull request by njlr**
This is a fix for https://github.com/unindented/custom-immutable-matchers/issues/39

Can you advise on your test & build process? 

A `CONTRIBUTING.md` would be very helpful.

### CODE REVIEW

**Comment by AlanFoster** [comment]
Ran into this problem when migrating a large code base from Jasmine to Jest 22 :)
Would be good to get this in if possible 🎉

**Comment by jamesmoss** [comment]
@unindented Any progress on merging this? We're waiting on this before we can upgrade to a newer version of Jest.

**Comment by unindented** [comment]
Shit, I wasn't subscribed to the repo, so I didn't get the email notification. Sorry @njlr!

File: packages/jest-immutable-matchers/package.json [hunk 1]
```diff
@@ -29,13 +29,13 @@
   },
   "peerDependencies": {
     "immutable": "*",
-    "jest": "*"
+    "jest": "^21"
```
Comment (unindented) [inline packages/jest-immutable-matchers/package.json]: Can we make this `>=`? We're already on `22`.

Comment (unindented) [reply packages/jest-immutable-matchers/package.json]: Don't worry, I'll merge and add it myself.

**Comment by unindented** [comment]
@njlr @AlanFoster @jamesmoss Just published `v2.0.0` with these changes. Sorry all!

**Pull request merged**